websites/M/Moopa/metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "338295424479789057",
+		"name": "factiven"
+	},
+	"service": "Moopa",
+	"description": {
+		"en": "Moopa is a free Anime and Manga streaming website with AniList integration where you can easily track your progress with a single click of a button."
+	},
+	"url": "moopa.live",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/BzZw6aQ.png",
```

Comment:
Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).